### ISSUE TITLE
insurance raise rates

### ISSUE THREAD

**Issue by yoel123** [yoel123/java-text-based-space-station-manger-sim#43]
basically whenever something bad happens, thers a chance the insurance rate will go up, if a lot of bad stuff happen the insurance rate will go up even more.

very easy to implement, you will need something to count the amount of bad stuff happening and then every turn run a method that checks if the rate shod go up by a random chance and the bad stuff counter shod affect this random variable.

also, reset the bad stuff counter every couple of turns. just to be fair.

thers sod be a way to request a rate reduction, but only when bed stuff happened counter is low or 0.

basically the insurance system is done, this is just another nice easy feature contributers can do.

**Comment by Keenal** [comment]
I'd like to give this a try for Hacktoberfest! (In order for it to get counted, the maintainer would have to give this issue a label 'Hacktoberfest').

**Comment by yoel123** [comment]
contact me via the contact form first please (i will add the Hacktoberfest label later).

**Comment by Keenal** [comment]
Contacted you!

**Comment by yoel123** [comment]
did you get my mail?

**Comment by Keenal** [comment]
I did not!
